docs/Changelog.md

```diff
@@ -14466,7 +14466,7 @@ This version of the operator has been available since version 12 of the default
 
 <dl>
 <dt><tt>auto_pad</tt> : string (default is NOTSET)</dt>
-<dd>auto_pad must be either NOTSET, SAME_UPPER, SAME_LOWER or VALID. Where default value is NOTSET, which means explicit padding is used. SAME_UPPER or SAME_LOWER mean pad the input so that `output_shape[i] = ceil(input_shape[i] / strides[i])` for each axis `i`. The padding is split between the two sides equally or almost equally (depending on whether it is even or odd). In case the padding is an odd number, the extra padding is added at the end for SAME_UPPER and at the beginning for SAME_LOWER.</dd>
+<dd>auto_pad must be either NOTSET, SAME_UPPER, SAME_LOWER or VALID. Where default value is NOTSET, which means explicit padding is used. SAME_UPPER or SAME_LOWER mean pad the input so that the output spatial size match the input.In case of odd number add the extra padding at the end for SAME_UPPER and at the beginning for SAME_LOWER. VALID mean no padding.</dd>
```

Comment:
Reading `the output spatial size match the input` (in my broken English) sounds a bit like "input size is equal to output size". Is the original: `output_shape[i] = ceil(input_shape[i] / strides[i])` for each axis `i` no longer valid?

Reply:
I do not actually remember changing that at all, I am not sure why it appeared in the changelog. I'll look into it.

Reply:
Fixed, there was a wrong reference to a docstring in `old.cc`. Thanks for letting me know!